### DISCUSSION TITLE
Unused files?

### DISCUSSION THREAD

**Discussion by scarf005** [cataclysmbn/Cataclysm-BN#2086] [Q&A] [answered]
Are these files used?

- .appveyor.yml
- .lgtm.yml
- org.cataclysmdda.CataclysmDDA.json
- cataclysm-dda.appdata.xml
- HOW_YOU_CAN_HELP.md

**Comment by olanti-p** [comment]
> .appveyor.yml

Nope, we don't use Appveyor

> .lgtm.yml

Nope, we don't use LGTM

> org.cataclysmdda.CataclysmDDA.json

Nope, we don't support flatpak, though if someone fixed it to work for BN it could be kept

> cataclysm-dda.appdata.xml

From what I can tell the whole `data/xdg/` directory is only used for flatpak, so same solution here. Exception is `cataclysm-dda.svg`, which is also used by `snapcraft.yaml`, but snapcraft also needs to be updated for BN

> HOW_YOU_CAN_HELP.md

Nope, we have https://github.com/cataclysmbnteam/Cataclysm-BN/wiki/Want-to-help%3F

**Reply by scarf005** [reply]
+ I also think travis.yml is unused.

should we leave them as-is?

**Reply by olanti-p** [reply]
Yes, Travis is also unused.

I say we remove them, because new people may think these files work, while in fact they don't.

Couple more, possibly broken, definitely could use a look:
- Cataclysm-DDA.sublime-project (Sublime Text project)
- CataclysmWin.cbp (Code::Blocks project)

**Comment by scarf005** [comment, marked as answer]
addressed with #2213
